src/DssDirectDepositAaveDai.sol

```diff
@@ -381,20 +467,21 @@ contract DssDirectDepositAaveDai {
     function cull() external {
         require(vat.live() == 1, "DssDirectDepositAaveDai/no-cull-during-shutdown");
         require(live == 0, "DssDirectDepositAaveDai/live");
-        require(_add(tic, tau) <= block.timestamp, "DssDirectDepositAaveDai/early-cull");
+        require(_add(tic, tau) <= block.timestamp || wards[msg.sender] == 1, "DssDirectDepositAaveDai/early-cull");
         require(culled == 0, "DssDirectDepositAaveDai/already-culled");
 
         (uint256 ink, uint256 art) = vat.urns(ilk, address(this));
         require(ink <= 2 ** 255, "DssDirectDepositAaveDai/overflow");
         require(art <= 2 ** 255, "DssDirectDepositAaveDai/overflow");
-        vat.grab(ilk, address(this), address(this), address(chainlog.getAddress("MCD_VOW")), -int256(ink), -int256(art));
+        vat.grab(ilk, address(this), address(this), chainlog.getAddress("MCD_VOW"), -int256(ink), -int256(art));
+
         culled = 1;
         emit Cull();
     }
 
     // --- Emergency Quit Everything ---
     function quit(address who) external auth {
-        require(live == 0, "DssDirectDepositAaveDai/live");
+        require(vat.live() == 1, "DssDirectDepositAaveDai/no-quit-during-shutdown");
```

Comment:
`live` is changed to `0` in `cull()` which would be required for this to be called before. Just want to clarify the intent to change the function here.

Reply:
I think this might have been a mistake on the original PR. You should be able to quit at any time during non-ES operation.